trestle/core/control_interface.py

```diff
@@ -518,10 +512,10 @@ def is_withdrawn(control: cat.Control) -> bool:
 
         This is determined by property with name 'status' with value 'Withdrawn'.
         """
-        for prop in as_list(control.props):
-            if prop.name and prop.value:
-                if prop.name.lower().strip() == 'status' and prop.value.lower().strip() == 'withdrawn':
-                    return True
+        for _ in as_filtered_list(
+                control.props,
+                lambda p: strip_lower_equals(p.name, 'status') and strip_lower_equals(p.value, 'withdrawn')):
+            return True
```

Comment:
if as_filtered_list(.... -> return True?

Reply:
I think it's ok this way because it allows a short-circuit exit on the first hit.  I don't think just using `if` would avoid evaluating the entire array if there are multiple hits.

Reply:
I meant just simply checking if the resulting array is empty or not would be the same?

Reply:
If you have 10 million items and the first is a hit then the loop i have will immediately exit and I don't think the `if` approach will do that.  I believe it will evaluate the entire list.  If that is incorrect it would be good to know.